package.json

```diff
@@ -28,23 +28,22 @@
     "postcss-preset-env": "^7.6.0",
     "prettier": "^2.3.2",
     "tailwindcss": "^3.0.24",
-    "typescript": "^4.7.4",
+    "typescript": "^4.7.2",
     "vite": "^2.9.0",
     "vitest": "^0.15.2"
   },
   "prettier": "@shopify/prettier-config",
   "dependencies": {
     "@headlessui/react": "^1.6.4",
     "@heroicons/react": "^1.0.6",
-    "@okendo/shopify-hydrogen": "^1.0.3",
-    "@shopify/hydrogen": "^1.0.0",
+    "@okendo/shopify-hydrogen": "file:okendo-shopify-hydrogen-1.0.3.tgz",
```

Comment:
Will `1.0.3` still be the right version?

Reply:
Nah it'll need to be updated once the npm package has been published.